With static graph enabled, relative path isn't resolved correctly

```
PS C:\git\runtime3> dotnet build src\libraries\System.Runtime\ref\System.Runtime.csproj /bl
Microsoft (R) Build Engine version 16.8.0+126527ff1 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

Some command line switches were read from the auto-response file "MSBuild.rsp". To disable this file, use the "-noAutoResponse" switch.
C:\Program Files\dotnet\sdk\5.0.100\MSBuild.dll -consoleloggerparameters:Summary -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Program Files\dotnet\sdk\5.0.100\dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,C:\Program Files\dotnet\sdk\5.0.100\dotnet.dll -graph -maxcpucount -restore -verbosity:m /bl src\libraries\System.Runtime\ref\System.Runtime.csproj
  Determining projects to restore...
  All projects are up-to-date for restore.
C:\git\runtime3\src\libraries\System.Runtime\ref\src\libraries\System.Runtime\ref\System.Runtime.csproj : error MSB4025: The project file could not be loaded. Could not find a part of the path 'C:\git\runtime3\src\libraries\System.Runtime\ref\src\libraries\System.Runtime\ref\System.Runtime.csproj'.

Build FAILED.

C:\git\runtime3\src\libraries\System.Runtime\ref\src\libraries\System.Runtime\ref\System.Runtime.csproj : error MSB4025: The project file could not be loaded. Could not find a part of the path 'C:\git\runtime3\src\libraries\System.Runtime\ref\src\libraries\System.Runtime\ref\System.Runtime.csproj'.
    0 Warning(s)
    1 Error(s)
```

Using the 5.0.100 SDK. Looks like msbuild concatenates the relative path passed in with the path that resolves to the project file.

cc @rainersigwald @cdmihai

--- a/src/Build/Graph/GraphBuilder.cs
+++ b/src/Build/Graph/GraphBuilder.cs
@@ -283,7 +283,7 @@ namespace Microsoft.Build.Graph
                 valueComparer: StringComparer.OrdinalIgnoreCase,
                 items: solutionEntryPoint.GlobalProperties ?? ImmutableDictionary<string, string>.Empty);
 
-            var solution = SolutionFile.Parse(FileUtilities.NormalizePath(solutionEntryPoint.ProjectFile));
+            var solution = SolutionFile.Parse(solutionEntryPoint.ProjectFile);
 
             if (solution.SolutionParserWarnings.Count != 0 || solution.SolutionParserErrorCodes.Count != 0)
             {
@@ -410,7 +410,7 @@ namespace Microsoft.Build.Graph
 
                     AddGraphBuildGlobalVariable(globalPropertyDictionary);
 
-                    var configurationMetadata = new ConfigurationMetadata(FileUtilities.NormalizePath(entryPoint.ProjectFile), globalPropertyDictionary);
+                    var configurationMetadata = new ConfigurationMetadata(entryPoint.ProjectFile, globalPropertyDictionary);
                     entryPointConfigurationMetadata.Add(configurationMetadata);
                 }
 

--- a/src/Build/Graph/ProjectGraphEntryPoint.cs
+++ b/src/Build/Graph/ProjectGraphEntryPoint.cs
@@ -31,12 +31,12 @@ namespace Microsoft.Build.Graph
         {
             ErrorUtilities.VerifyThrowArgumentLength(projectFile, nameof(projectFile));
 
-            ProjectFile = projectFile;
+            ProjectFile = FileUtilities.NormalizePath(projectFile);
             GlobalProperties = globalProperties;
         }
 
         /// <summary>
-        /// Gets the project file to use for this entry point.
+        /// Gets the full path to the project file to use for this entry point.
         /// </summary>
         public string ProjectFile { get; }
 

